### ISSUE TITLE
Fix composable stable create pool v2 flaky test

### ISSUE THREAD

**Issue by brunoguerios** [balancer/b-sdk#548]
It's failing intermittently on CI

Note: likely related to us relying on finding event on receipt logs to assert the pool was properly created? Can we leverage the `create` function output? All different pool factories return the same output for the create function, which is the pool address.

**Comment by brunoguerios** [comment]
I'm not able to reproduce that locally and I'm not really sure on how to solve it.
We can't apply the suggested solution of relying on the `create` function output because we build transaction `data` that is sent using `sendTransaction`. From there, it seems that `transactionReceipt` logs are indeed the only way to get the `address` from the recently created pool.

**Comment by brunoguerios** [comment]
I was under the assumption that the problem was within `@viem/anvil` dev dependency we use to run anvil forks, based on the fact that it doesn't fail on a tenderly fork.
On top of that @viem/anvil has been discontinued in favor of `prool` also developed by `wevm` team.
I tried replacing it but I wasn't able to import prool into b-sdk. There was a conflicting configuration.
The suggested solution was to add `type: module` to b-sdk `tsconfig` but that would have many side effects that I'm not able to evaluate at the moment.
Sharing here for context in case anyone decides to pursue that in the future.

**Comment by brunoguerios** [comment]
next attempt: evaluate tests with old setup, by manually running a fork node and testing against it

**Comment by brunoguerios** [comment]
old setup doesn't fix this - maybe caused by anvil? 
we could try replacing by hardhat to investigate, but not sure we'd like to add that dependency
what do you think?

**Comment by johngrantuk** [comment]
> old setup doesn't fix this - maybe caused by anvil? we could try replacing by hardhat to investigate, but not sure we'd like to add that dependency what do you think?

Lets skip it so we can move on. We'll just have to keep monitoring for any changes to that part of code (which should be very rare). Let me know if you disagree otherwise can you open a PR with the changes?

**Comment by brunoguerios** [comment]
I agree we should simply skip these tests for now. Since the PR intention was to re-enable them, there's nothing to be done. Closing as won't fix 🚫
